resources.html

```diff
@@ -56,4 +56,38 @@ <h4 class="card-title">{{blog.title}}</h4>
             </div>
         {% endfor %}
     </div>
+
+    <div class="resources__block events">
+        <h1>Events</h1>
+
+        {% for event in site.data.events %}
+        <div class="row">
+            <div class="col-12 mt-3">
+                <div class="card">
+                    <div class="row">
+
+                        <div class="col-md-5 col-lg-4">
+                            <img class="card-image" src="assets/images/resources/events/panel_discussion_mentors.jpg" alt="Panel Discussion “What makes a good mentor?”">
```

Comment:
this image link needs to be dynamic from the yml file, for example: 

```
image: assets/images/resources/events/panel_discussion_mentors.jpg 
imageAlt: "Panel Discussion: What makes a good mentor?"
```

and here you would add:
```suggestion
                            <img class="card-image" src="{{event.image}}" alt="{{event.imageAlt}}">
```